app/src/js/reducers/reconciliation-reports.js

```diff
@@ -45,10 +45,7 @@ export default createReducer(initialState, {
     };
   },
   [RECONCILIATIONS]: (state, action) => {
-    // response.results is a array of string filenames
-    const reports = action.data.results.map((filename) => ({
-      reconciliationReportName: filename,
-    }));
+    const reports = action.data.results;
```

Comment:
Can you write a simple reducer test for this one?

Reply:
Done